release/Makefile

```diff
@@ -0,0 +1,105 @@
+# Needed for glob expansion.
+SHELL = /bin/bash
+.SHELLFLAGS = -O extglob -c
+
+ifneq ($(DRY_RUN),)
+TAG ?= $(GIT_VERSION)
+endif
+
+.PHONY: all
+all: release
+
+.PHONY: check_tag
+check_tag:
+ifeq ($(TAG),)
+	$(error "TAG must be set to a release tag")
+endif
+
+.PHONY: changelog
+changelog: check_tag ## Generate the changelog.
+	mkdir -p changelog/generated
```

Comment:
What is this directory for?

Reply:
Another thing I forgot to remove. This will be in #4034.